llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -1262,9 +1268,23 @@ VPlan *VPlan::duplicate() {
   assert(Old2NewVPValues.contains(TripCount) &&
          "TripCount must have been added to Old2NewVPValues");
   NewPlan->TripCount = Old2NewVPValues[TripCount];
+
+  for (unsigned I = CreatedBlockSize; I != CreatedBlocks.size(); ++I)
```

Comment:
`for (unsigned I : seq<unsigned>(CreatedBlockSize, CreatedBlocks.size()))`

Reply:
done thanks